services/ui-src/src/components/modals/AddEditReportModal.tsx

```diff
@@ -32,13 +32,40 @@ export const AddEditReportModal = ({
   // WP report payload
   const prepareWpPayload = () => {
     const submissionName = "Work Plan";
+    // static entities
+    const targetPopulation = [
+      {
+        id: "2Vd02CVUtKgBETwqzDXpSIhi",
```

Comment:
for my own knowledge, where do these id's come from? :-)

Reply:
of course! so these are randomly generated KSUIDs, I've been using this website for it: https://ksuid.net/